File: Triatlon-CS/Services/IServices.cs
using Triatlon_CSharp.Domain;

namespace Services
{
    public interface IServices
    {
        User login(User user, IObserver client);
        void addResult(Result result);
        void logout(User user, IObserver client);
        Participant[] getParticipantsByScore(int idStage);
        ParticipantDTO[] getParticipantsAndScore();
        Participant findParticipant(int id);
        User findUser(int id);
        User findUserByUsername(User user);
    }
}

File: Triatlon-Java/ChatServices/src/main/java/triathlon/services/IServices.java
package triathlon.services;

import triathlon.model.User;
import triathlon.model.Participant;
import triathlon.model.ParticipantDTO;
import triathlon.model.Result;


public interface IServices {
    User login(User user, IObserver client) throws TriatlonException;
    void addResult(Result result) throws TriatlonException;
    void logout(User user, IObserver client) throws TriatlonException;
    Participant[] getParticipantsByScore(Integer idStage) throws TriatlonException;
    ParticipantDTO[] getParticipantsAndScore() throws TriatlonException;

//    Participant[] getParticipantsByScore(Integer idStage) throws TriatlonException;
//    ParticipantDTO[] getParticipantsAndScore() throws TriatlonException;
}
